vundo-diff.el

```diff
@@ -158,6 +158,20 @@ the original buffer name."
               (display-buffer dbuf)))
         (kill-buffer mrkbuf)))))
 
+;;;###autoload
+(defun vundo-diff-quit ()
+  "Close the diff buffer & window, when found."
+  (interactive)
+  (when vundo--orig-buffer
+    (let* ((orig vundo--orig-buffer)
+           (oname (buffer-name orig))
+           (dbuf (get-buffer (concat "*vundo-diff-" oname "*"))))
+      (when dbuf
+        (let ((dbuf-window (get-buffer-window dbuf)))
+          (when dbuf-window
```

Comment:
This may be too aggressive.  We don't know whether the vundo-diff window was a new window dedicated to vundo, which would depend on the user's `display-buffer-alist` and related config.

Reply:
Ah, I didn't think of that, in that case vundo-diff it's self could surely track if it created a new window when invoked, if so, remove that window.

Reply:
Committed a fix that only closes the window if displaying the buffer crated it.